integration/commands_administration_test.go

```diff
@@ -159,24 +160,112 @@ func TestCommandsAdministrationListDatabases(t *testing.T) {
 	db := collection.Database()
 	name := db.Name()
 
-	actual, err := db.Client().ListDatabases(ctx, bson.D{{"name", name}})
-	require.NoError(t, err)
-	require.Len(t, actual.Databases, 1)
+	t.Run("Exists", func(t *testing.T) {
+		actual, err := db.Client().ListDatabases(ctx, bson.D{{Key: "name", Value: name}})
+		require.NoError(t, err)
+		require.Len(t, actual.Databases, 1)
+
+		expected := mongo.ListDatabasesResult{
+			Databases: []mongo.DatabaseSpecification{{
+				Name:       name,
+				SizeOnDisk: actual.Databases[0].SizeOnDisk,
+				Empty:      actual.Databases[0].Empty,
+			}},
+			TotalSize: actual.TotalSize,
+		}
 
-	expected := mongo.ListDatabasesResult{
-		Databases: []mongo.DatabaseSpecification{{
-			Name:       name,
-			SizeOnDisk: actual.Databases[0].SizeOnDisk,
-			Empty:      actual.Databases[0].Empty,
-		}},
-		TotalSize: actual.TotalSize,
-	}
+		assert.Equal(t, expected, actual)
+
+		assert.NotZero(t, actual.Databases[0].SizeOnDisk, "%s's SizeOnDisk should be non-zero", name)
+		assert.False(t, actual.Databases[0].Empty, "%s's Empty should be false", name)
+		assert.NotZero(t, actual.TotalSize, "TotalSize should be non-zero")
+	})
+
+	t.Run("ExistsNameOnly", func(t *testing.T) {
+		actual, err := db.Client().ListDatabases(ctx, bson.D{
+			{Key: "name", Value: name},
+		},
+			&options.ListDatabasesOptions{
+				NameOnly: pointer.To(true),
+			})
```

Comment:
That formatting is unlike anything we already use. Let's use `options.ListCollections().SetNameOnly(true)` – we use methods for options everywhere